packages/fxa-auth-server/lib/senders/subscription-templates/downloadSubscription.html

```diff
@@ -21,7 +21,7 @@ <h1 style="font-size: 1.1em; line-height: 1.5;">
 <tr style="page-break-before: always">
   <td align="left" valign="top" style="padding: 30px 30px 15px;">
     <p style="font-size: 0.9em; line-height: 1.8;">
-      Questions about your subscription? Our <a {{{supportLinkAttributes}}}>support team</a> is here to help you.
+      Questions about your subscription? Our <a href="{{{subscriptionSupportUrl}}}">support team</a> is here to help you.
```

Comment:
I'm presuming the strings are only going to be translated once we ship outside of the US?

Reply:
Affirmative